nep-7.mediawiki

```diff
@@ -0,0 +1,117 @@
+<pre>
+  NEP: 7.0
+  Title: URI Scheme
+  Author: Andrei <terentiev.andrei@gmail.com>, Apisit <apisit.toompakdee@gmail.com>
+  Type: Standard
+  Status: Draft
+  Created: 2018-01-15
+</pre>
+
+==Abstract==
+
+This NEP describes a URI standard for transfer of NEO native assets.  
+
+==Motivation==
+
+Currently, there is no standard URI for NEO clients to consume. Bitcoin implements a URI standard so that clicking links or scanning QR codes can easily implement a bitcoin payment. Similarly the transfer of native assets on the NEO chain should be just as simple.
+
+In the future this URI scheme should enable generic smart contract invocation. This would allow for transfer of NEP-5 tokens, but these extensions should be added on once the more basic URI standards are agreed upon
+
+==Specification==
+
+===Native Asset Transfer URI===
+
+A Native asset transfer would have the following URI. It describes the reciever address, assetID, and additional attributes that will be sent with the transaction. The attributes are presented in raw hex data.
+
+<code>NEO:<address>[?assetID=<assetID>][?amount=<amount>][?attributes=<key>=<value>]</code>
```

Comment:
I would prefer that the key/value pair be separated by another symbol. I think its good to follow querystring conventions, not sure if the decimal point in the amount will screw things up

Reply:
I believe this follows query string convention.
the decimal point in it is fine. Please see s